### ISSUE TITLE
a batch question about horovod

### ISSUE THREAD

**Issue by 652994331** [horovod/horovod#1648]
i used this horovod for a multigpu_pretrain on September, 2019. it's really good. recently i was thinking about a question about horovod. According to my understanding, horovod is a data split strategy multi-gpu training like tensorflow mirrored strategy. Which means, for example, i have 4 data_tfrecords files and i want to do a 4 gpus horovod pretrain, horovod gonna assign these 4 to 4 gpus, one on one.
So, if the command line is --train_batchsize =64, we are actually used 64*4 = 256 batchsize because of this strategy. Am I right guys? 

Thanks for your help, really appreciate

**Comment by tgaddair** [comment]
Hey @652994331, yes, this is correct.  Total batch size is <num_workers> * <batch_size_per_worker>.  This is why we need to scale the learning rate proportional to `hvd.size()`.

**Comment by 652994331** [comment]
@tgaddair got you, thanks so much

**Comment by 652994331** [comment]
@tgaddair  When I looked in to the code, i found the train_input_fn still use train_batch_size. if we want to use global_batch_size, we need to make sure report_loss parameter is TRUE, am i right? Thanks
